README.md

```diff
@@ -428,95 +428,20 @@ The Operator SDK CLI can generate CSV manifests via the following command:
 ```console
 $ operator-sdk olm-catalog gen-csv --csv-version 0.0.1
 ```
-For more details see the SDK's [CSV generation doc][csv_generation_doc].
-
-For the purpose of this guide, we will continue with this [predefined manifest][manifest_v1] file for the next steps. If you’d like, you can alter the image field within this manifest to reflect the image you built in previous steps, but it is unnecessary.
-
-### Deploy the Operator
 
-Deploying an operator is as simple as applying the operator’s CSV manifest to the desired namespace in the cluster.
+Then, after generate the CSV file will be required update it with the required information over the memcached-project which could not be done automatically by the tool. See an example of an [Memcached CSV][memcached_csv] file filled.
```

Comment:
```suggestion
Several fields must be updated after generating the CSV. See the CSV generation doc for a list of [required fields][csv-fields], and the memcached-operator [CSV][memcached_csv] for an example of a complete CSV.
```

And make a link ref below for `csv-fields: https://github.com/operator-framework/operator-sdk/blob/master/doc/user/olm-catalog/generating-a-csv.md#csv-fields`